File: GTTS/INL.VettingService.Models/GetPostVettingType_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class GetPostVettingType_Item :IGetPostVettingType_Item
    {
        public int PostID { get; set; }
        public int VettingTypeID { get; set; }
        public string Code { get; set; }
        public string Description { get; set; }
        public bool IsActive { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-post-vetting-type_item.ts


import { IGetPostVettingType_Item } from './iget-post-vetting-type_item';

export class GetPostVettingType_Item implements IGetPostVettingType_Item {
  
	public PostID: number = 0;
	public VettingTypeID: number = 0;
	public Code: string = "";
	public Description: string = "";
	public IsActive: boolean = false;
  
}


